test/test_plugin_helper.rb

```diff
@@ -15,3 +15,9 @@ def disable_webpack
     Webpack::Rails::Manifest.stubs(:asset_paths).returns([])
   end
 end
+
+def theme_fixture_file(filename)
+  File.join(
+    ForemanThemeSatellite::Engine.root, 'test', 'fixtures', filename
+  )
```

Comment:
FYI: `ForemanThemeSatellite::Engine.root` is a [Pathname](https://docs.ruby-lang.org/en/3.2/Pathname.html) so you can do
```suggestion
  ForemanThemeSatellite::Engine.root.join('test', 'fixtures', filename)
```

Reply:
https://github.com/RedHatSatellite/foreman_theme_satellite/pull/18